search/tests/test_course_discovery.py

```diff
@@ -492,6 +492,8 @@ class TestMeilisearchCourseDiscoverySearch(TestCase, SearcherMixin):
     Integration tests using real Meilisearch engine.
     """
 
+    meilisearch_client = get_meilisearch_client()
```

Comment:
Could you provide some context as to why this is being moved up here instead of following the previous pattern of calling `get_meilisearch_client()` in `tearDown` and `wait_for_meilisearch_indexing`?